openapi-v1.yaml

```diff
@@ -8242,6 +8260,11 @@ paths:
           description: pagination limit
           type: integer
           required: false
+        - name: type
```

Comment:
Maybe `group_type` instead of `type` for consistency? `type` usually refers to array type.